### DISCUSSION TITLE
Language hierarchy?

### DISCUSSION THREAD

**Discussion by RexGhidorah** [anidl/multi-downloader-nx#676] [Q&A] [answered]
Hello!
No matter how I select the languages, always the final mkv file they will have them like this:
-Japanese
-English
-Spanish

Is there a way to change this, like, English or Spanish over Japanese?
I mean, there is always the possibility of downloading it in parts but that just takes time....

Config:
![image](https://github.com/anidl/multi-downloader-nx/assets/168611490/6e3d93ca-02dd-4229-aace-f4cfce0f9a98)

MKVToolnix showing what i described:
![image](https://github.com/anidl/multi-downloader-nx/assets/168611490/c039175c-49b7-4716-8874-a2b36409b7fc)

**Comment by AnimeDL** [comment, marked as answer]
Languages are currently defined in the order of which the streaming service has them, I've been meaning to change it so that it is ordered to the order of the option but I haven't gotten around to it. Consider opening a feature request
